racetrack_client/racetrack_client/plugin/install.py

```diff
@@ -67,6 +69,62 @@ def uninstall_plugin(
     logger.info(f'Plugin {plugin_name} {plugin_version} has been uninstalled from {lifecycle_url}')
 
 
+def download_installed_plugin_version(
+        lifecycle_url: Optional[str],
+        out_dir: Optional[str | Path],
+        out_filename: Optional[str | Path ],
```

Comment:
You can pass the `out_filename` parameter, but it's always overwritten later on. I think, just as well, we could remove this parameter.